test/functionality/dataflow/processing-of-elements/atomic/atomic-tests.ts

```diff
@@ -86,7 +86,7 @@ describe('Atomic (dataflow information)', withShell((shell) => {
 				.addEdge('2', '1', EdgeType.Reads, 'always')
 				.addEdge('5', '4', EdgeType.Reads, 'always')
 		)
-		assertDataflow('assign on access', shell,
+		assertDataflow(label('assign on access', 'local-left-assignment', 'single-bracket-access'), shell,
```

Comment:
`normal-name` (or name-normal :D not sure) would fit as well :) you can give the number as well.